HunterPie.Core/Game/Data/AbnormalityData.cs

```diff
@@ -68,13 +69,15 @@ private static void LoadAbnormalities()
                 Id = BuildAbnormalityId(id, group),
                 Name = name,
                 Icon = icon,
+                CompareOpr = withValueNot != null ? (int)CompareType.WithValueNot : (int)CompareType.WithValue,
```

Comment:
Why are you hard casting an enum to `int`? You can just define `CompareOpr` as `CompareType`.